cloudify_vsphere/contentlibrary/deployment.py

```diff
@@ -36,6 +37,13 @@
 def create(ctx, connection_config, library_name, template_name, target,
            deployment_spec):
     runtime_properties = ctx.instance.runtime_properties
+
+    if runtime_properties.get(VSPHERE_SERVER_ID):
+        ctx.logger.info("VM {vm_name} deployed with id: {vm_id}".format(
```

Comment:
Shouldn't this say "VM template {vm_name}" rather than "VM {vm_name}", since it's part of the content library?

Reply:
I have replaced message to "VM template deployed with id: {vm_id} and name: {vm_name}"